client/packages/common/src/types/schema.ts

```diff
@@ -3557,7 +3557,7 @@ export enum InvoiceNodeStatus {
    * Outbound Shipment: Becomes not editable
    * Inbound Shipment: For inter store stock transfers an inbound Shipment
    * becomes editable when this status is set as a result of corresponding
-   * outbound Shipment being chagned to shipped (this is similar to New status)
+   * outbound Shipment being changed to shipped (this is similar to New status)
```

Comment:
I think need another yarn generate too, this file should show the item direction changes?

Reply:
Will do! I wasn't sure if the yarn generate is done for the backend PR or the frontend trying to consume.

Reply:
d6d85dbc4b94